Assignment Table Styling

christinem opened the pull request:
This PR restyles the assignment and dashboard table on both the student and admin/ta side so peer review assignments are now grouped together with their original assignments in the table.

david-yz-liu commented on app/views/assignments/_assignment_summary.html.erb:
```diff
@@ -2,8 +2,8 @@
       <%= link_to I18n.t(:assignment_link,
-                         short_id: assignment.short_identifier,
-                         description: assignment.description),
+                         short_id: assignment.is_peer_review? ? assignment.short_identifier[0,2] : assignment.short_identifier,
```
Don't ever use the peer review short_identifier. Use the parent assignment.

christinem commented:
@david-yz-liu Fixed the merge conflicts in this branch.

david-yz-liu commented:
@christinem the peer review tab for students needs to be hidden when the PR assignment is hidden.

david-yz-liu commented:
Also the peer review short identifier appears in other tabs still, including the Reviews table.

christinem commented:
@david-yz-liu I fixed the peer review tab comment. As for the short identifier, I fixed most of those in #2704. The only one I didn't change yet is the short identifier on the edit assignment page, as I'm not sure whether that one should be changed.
